src/scripts/site.js

```diff
@@ -381,14 +387,13 @@ var AppInfo = {};
                 return self.ResizeObserver;
             });
         } else {
-            define("ResizeObserver", ["thirdparty/resize-observer-polyfill/ResizeObserver"], returnFirstDependency);
+            define("ResizeObserver", [getBowerPath() + "/resize-observer-polyfill/ResizeObserver"], returnFirstDependency);
         }
     }
 
     function initRequireWithBrowser(browser) {
         var bowerPath = getBowerPath();
-        var apiClientBowerPath = bowerPath + "/apiclient";
```

Comment:
why remove that and copy-paste `/apiclient` all over the place?

Reply:
I did this because the goal is to eventually remove `bowerPath` entirely and pull the dependencies in during the build, and consolidating all of the libraries to remove under `bowerPath` helps with searching for now.